lib/flipper/ui/views/features.erb

```diff
@@ -28,22 +28,50 @@
           </th>
           <th class="col">Feature</th>
           <th class="col">Enabled Gates</th>
+          <% if flipper.adapter.adapter.name.eql? :redis %>
+            <th class="col">Gate Timestamp</th>
+          <% end %>
         </tr>
       </thead>
       <tbody>
-        <% @features.each do |feature| %>
-          <tr class="d-flex">
-            <td class="col-1">
-              <span class="octicon octicon-squirrel <%= feature.color_class %>"></span>
-            </td>
-            <td class="col">
-              <a href="<%= "#{script_name}/features/#{feature.key}" %>">
-                <%= feature.key %></a>
-            </td>
-            <td class="col">
-              <%= feature.pretty_enabled_gate_names %>
-            </td>
-          </tr>
+        <% if false %>
```

Comment:
This is where I was hoping to have the toggle to turn Time Sorting on/off. I wasn't exactly sure how to go about doing so within an erb file. Would love some suggestions!